ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+
+import { useAPI } from '../../../api/useAPI';
+import PropTypes from 'prop-types';
+import { Link, useParams, useLocation } from '@reach/router';
+import { getServicesEndpoint } from '../../../api/paths';
+
+const propTypes = {
+  service: PropTypes.string.isRequired,
+  finding: PropTypes.string.isRequired,
+};
+
+const Resources = (props) => {
+  const { service } = props;
+  const { data: categories, loading } = useAPI(getServicesEndpoint(), []);
+  const { pathname } = useLocation();
+  const params = useParams();
+
+  if (loading) return null;
+
+  let resourceName = '';
+
+  for (let category of categories) {
+    for (let service of category.services) {
+      for (let res of service.resources) {
+        if (res.id === params.resource) resourceName = res.name;
```

Comment:
using `Array.find` would be better here, something like: `resourceName = service.resources.find(res => res.id === params.resource).name;`

Reply:
it's slightly more complicated than I thought to make it work, keep the one you prefer.
![image](https://user-images.githubusercontent.com/26723367/111839016-8754f880-88d0-11eb-82ad-51b50fb42cd1.png)

Reply:
Found a much cleaner way of doing it 🙂
```js
const services = flatten(categories.map(c => c.services));
const resources = flatten(services.map(s => s.resources));
const resource = resources.find((res) => res.id === params.resource);
```

Reply:
i like it, much better